.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'BUG: '
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Environment Happening In**
+- [ ] Production
+- [ ] QA
+- [ ] Dev
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A clear and concise description of what actually happened.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Related Tickets**
+List out any user story or bug ticket(s) related to this bug. 
+
+**Desktop (please complete the following information):**
+- Product: [e.g. axe-core] 
```

Comment:
shouldn't this be axe-core-maven-html this the issue would be opened for this repo?